src/main/java/uk/gov/pay/api/model/CreatePaymentRequest.java

```diff
@@ -13,19 +13,22 @@
     public static final String REFERENCE_FIELD_NAME = "reference";
     public static final String DESCRIPTION_FIELD_NAME = "description";
     public static final String AGREEMENT_ID_FIELD_NAME = "agreement_id";
+    public static final String LANGUAGE_FIELD_NAME = "language";
 
-    private int amount;
-    private String returnUrl;
-    private String reference;
-    private String description;
-    private String agreementId;
+    private final int amount;
+    private final String returnUrl;
+    private final String reference;
+    private final String description;
+    private final String agreementId;
+    private final String language;
```

Comment:
Isn't there a language enum in common library for this? Would it be better to use that than String?

Reply:
That’s coming in the next PR!

There’s two levels of checking of the payload. The first is done as part of the JSON deseralisation: it basically just checks the JSON has fields with the right names and that each field is of the appropriate basic JSON type (string etc.).

Later validation (coming in the next PR) does more semantic checks, such as that the URLs are valid, the amount is within the appropriate range etc. This is where we’ll check the language is actually a supported code and where we’ll start using the enum.